docs/api-inference/tasks/chat_completion.md

```diff
@@ -1,3 +1,16 @@
+<!---
+This markdown file has been generated from a script. Please do not edit it directly.
+For more details, check out:
```

Comment:
maybe also add the task page metadata as that includes the models for the examples

Reply:
added in https://github.com/huggingface/hub-docs/pull/1400/commits/143367a3c13bde5b0d8b080b508793ccb7e4cc23